### PR TITLE
Checkstyle fixes for async-rest sample.

### PR DESCRIPTION
**Pull request by tswast**
Tested manually by deploying to http://async-rest-test-dot-java-docs-samples-swast-1.appspot.com/.

When doing so, I noticed that the `mvn gcloud:deploy` command would not
run, so I updated the POM to the latest version of the gcloud maven
plugin, too.

Depends on #115.
Fixes https://github.com/GoogleCloudPlatform/java-docs-samples/issues/61.

### CODE REVIEW

**Comment by tswast** [comment]
Depends on https://github.com/GoogleCloudPlatform/java-docs-samples/pull/115.

**Comment by tswast** [comment]
Fixes https://github.com/GoogleCloudPlatform/java-docs-samples/issues/61.

**Comment by tswast** [comment]
Build is failing due to https://github.com/GoogleCloudPlatform/java-docs-samples/issues/118. New monitoring sample is failing checkstyle.

File: java-repo-tools [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 5c9a3169e1ee8fd471c0fd7501b50e1b2bc16773
```
Comment (lesv) [inline java-repo-tools]: We should document this somewhere - like our internal page.

**Comment by lesv** [comment]
LGTM

File: java-repo-tools (hunk 1 above)
Comment (tswast) [inline java-repo-tools]: Sounds good. I can do that. I also think while we're in the business of auto-updating Maven plugins with Jenkins, we can consider auto-updating submodule commit numbers, too.

**Pull request merged**